### PR TITLE
Add ya2d, dbglogger, bzip2 and mini18n

### PR DESCRIPTION
**Pull request by rhynec**
Tested today, and building them as part of the main script works properly. They are also set as being executable, so there shouldn't be any issue with permissions. The dgblogger script could be modified to also build bzip2.

### CODE REVIEW

**Comment by rhynec** [comment]
Added bzip2 and mini18n to library scripts. The new mini18n script is added to allow for PKGi compilation straight from setup. I also created a branch which only provides the libraries missing from the 2020/07/17 pre-built toolchain: https://github.com/rhynec/ps3libraries/tree/extra-install

**Comment by bucanero** [comment]
I totally missed this PR. I'll review it

File: scripts/035-bzip2_dbglogger.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -e
+#
+# bzip2 by cadcdev
+# dbglogger by Bucanero
+
+## Download the source code.
+wget --no-check-certificate https://github.com/bucanero/psl1ght-libs/tarball/master -O psl1ght-libs.tar.gz
```
Comment (bucanero) [inline scripts/035-bzip2_dbglogger.sh:7]: Now there's a new repo for the `dbglogger` library.

Can you update the script so it downloads from https://github.com/bucanero/dbglogger ?

**Review by bucanero — CHANGES_REQUESTED** [review]
Needs new link for `dbglogger`

**Comment by rhynec** [comment]
Hey @bucanero!

I deleted my fork a while back, so it was easier for me to re-fork and merge the changes manually. I'll open a separate PR.

**Pull request closed without merging**